File: Jarvis.Web/MG.Jarvis.Api.Extranet/ViewModel/HotelPaymentMethodRelationViewModel.cs
namespace MG.Jarvis.Api.Extranet.ViewModel
{
    public class HotelPaymentMethodRelationViewModel : BaseViewModel
    {
        public int HotelPaymentMethodRelationId { get; set; }
        public int PaymentMethodId { get; set; }
        public int CurrencyId { get; set; }
        public int RateTypeId { get; set; }
        public bool IsDeleted { get; set; }
    }
}

File: Jarvis.Web/MG.Jarvis.Web/ClientApp/src/app/extranet/viewmodel/hoteldetailsviewmodel/hotelpaymentmethodrelationviewmodel.ts
import { ObjectState } from '../../../common/enums';
import { IObjectWithState } from '../../../common/iobjectwithstate';

export class HotelPaymentMethodRelationViewModel implements IObjectWithState {
    objectState: ObjectState;
    hotelPaymentMethodRelationId: number;
    paymentMethodId: number;
    currencyId: number;
    rateTypeId: number;
}
